SMLHelper/Logger.cs

```diff
@@ -1,36 +1,86 @@
 ﻿namespace SMLHelper.V2
 {
     using System;
+    using System.IO;
+
+    internal enum LogLevel
+    {
+        Debug = 0,
+        Warn = 1,
+        Error = 2,
+        Info = 3
+    }
 
     internal static class Logger
     {
-        internal static void Log(string text)
+        internal static LogLevel LoggingLevel { get; private set; }
+
+        internal static void Initialize()
+        {
+            string configPath = "./QMods/Modding Helper/loglevel.txt";
+
+            if(!File.Exists(configPath))
+            {
+                File.WriteAllText(configPath, "Warn");
+                LoggingLevel = LogLevel.Warn;
+
+                return;
+            }
+
+            string level = File.ReadAllText(configPath);
+
+            try
+            {
+                LoggingLevel = (LogLevel)Enum.Parse(typeof(LogLevel), level);
+
+                if (LoggingLevel > LogLevel.Error)
+                {
+                    LoggingLevel = LogLevel.Error;
+
+                    File.WriteAllText(configPath, "Error");
```

Comment:
Maybe also log this action?

Reply:
This if-block looks to be disallowing users from setting their log level to `Info`, and forcing `Error` as the minimum log level.  
If that was intentional, leave a comment (or log) to inform what happened and why.